api/src/test/java/org/openmrs/PersonTest.java

```diff
@@ -824,6 +824,15 @@ private void checkGetPersonNameResultForVoidedPerson(PersonName expectedPersonAd
 		
 		assertEquals(expectedPersonAddress, actualPersonName);
 	}
+
+	@Test
+	public void shouldSetDeadToTrueIfSetDeathdate() {
+
+				Person p = new Person();
+				Date deathDate = new Date();
+				p.setDeathDate(deathDate);
+				assert (p.getDead());
```

Comment:
It had something to do with this import "import static org.junit.Assert.assertTrue;" I am sorry I can't recall the exact issue. Since I could not remove the import as it may have affected other things in the class, I decided to use assert instead, and that cleared that particular issue